source-pendo/acmeCo/flow.yaml

```diff
@@ -0,0 +1,110 @@
+---
```

Comment:
We'd usually not have the `acmeCo` directory for connectors that aren't doing a capture snapshot test of the capture's output. Just leave `bindings: []` (empty) in the `test.flow.yaml`.